00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.02 Design Decisions/Content Lists.md

```diff
@@ -0,0 +1,40 @@
+## Sorting of lists, to aid readability
+
+### Decisions and conclusions
+
+- Sort on Alias, not Plugin ID
+- Avoid comment markers
+- Find sections based on headers - and maybe location files and folders
+- This is doable by GitHub action
+- Update scripts that generate lists, to sort in the first place - e.g. Uncategorised plugins
+
+### The Problem
+
+For example, I find it hard to glance at the category pages and see what I'm after.
+
+Compare these two:
+
+![task-plugins-sorted](attachments/images/task-plugins-sorted.png)
+
+### Suggestions
+
+For machine generated text, e.g. lists of plugins and themes in People pages, make the python code sort these lists by plugin/theme name.
+
+For human-edited lists, e.g. categories:
+
+Introduce a standard delimiter to mark lists that should be sorted by the text the link, case-insensitively, e.g. 
```

Comment:
> The text the link

Reply:
Fixed in 36d9bf90002f073c87f65accb9f36edba5d65be9